### ISSUE TITLE
compile warning

### ISSUE THREAD

**Issue by 9000h** [pesintta/vdr-plugin-vaapidevice#82]
```
cc -g -O3 -Wall -fPIC -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -I/usr/include/alsa -I/usr/include/x86_64-linux-gnu -DPLUGIN_NAME_I18N='"vaapidevice"' -D_GNU_SOURCE -DAV_INFO -DAV_INFO_TIME=3000	  -DGIT_REV='"15b6190"'  -g -W -Wall -Wextra -Winit-self -Wdeclaration-after-statement -Wno-unused-parameter -std=c99   -c -o audio.o audio.c
audio.c: In function ‘AudioInit’:
audio.c:2154:7: warning: this statement may fall through [-Wimplicit-fallthrough=]
    if (AudioChannelsInHw[4]) {
       ^
audio.c:2158:7: note: here
       case 4:
       ^~~~
audio.c:2159:7: warning: this statement may fall through [-Wimplicit-fallthrough=]
    if (AudioChannelsInHw[5]) {
       ^
audio.c:2163:7: note: here
       case 5:
       ^~~~
audio.c:2164:7: warning: this statement may fall through [-Wimplicit-fallthrough=]
    if (AudioChannelsInHw[6]) {
       ^
audio.c:2168:7: note: here
       case 6:
       ^~~~
audio.c:2169:7: warning: this statement may fall through [-Wimplicit-fallthrough=]
    if (AudioChannelsInHw[7]) {
       ^
audio.c:2173:7: note: here
       case 7:
       ^~~~
audio.c:2174:7: warning: this statement may fall through [-Wimplicit-fallthrough=]
    if (AudioChannelsInHw[8]) {
       ^
audio.c:2178:7: note: here
       case 8:
       ^~~~
```
sorry for bring this up again but it looks not nice at compile time

**Comment by rofafor** [comment]
What's the compiler and its' version you're using?

**Comment by 9000h** [comment]
should be GCC 7.3 from Ubuntu 18.04 but will check later today

**Comment by pesintta** [comment]
Fixed in PR #77
